apps/helptheducks.com/eslint.config.mjs

```diff
@@ -0,0 +1,42 @@
+import { fixupConfigRules } from "@eslint/compat";
```

Comment:
i think ideally we have a base eslint config that is shared b/w app js apps in the mono repo. then if any individual app requires extra options, they could extend it. same w/ prettier for js apps/packages..